tools/src/ToolsHelper.cc

```diff
@@ -80,10 +83,25 @@ bool parseOptions(int* argc, char** argv[], uint64_t* batchSize, orc::RowReaderO
         }
         break;
       }
+      case 'm' : {
+        *showMetrics = true;
+        break;
+      }
       default: break;
     }
   } while (opt != -1);
   *argc -= optind;
   *argv += optind;
   return true;
 }
+
+void printReaderMetrics(std::ostream& out, const orc::ReaderMetrics& metrics) {
+  out << "ReaderCount: " << metrics.ReaderCount << std::endl;
+  out << "ElapsedTimeUs: " << metrics.ReaderInclusiveLatencyUs << std::endl;
```

Comment:
nit: user may expect to read time in seconds which is much easier to interpret.

Reply:
The output of time metrics is uniformly changed to be in seconds.